sleap_nn/training/get_bin_files.py

```
@@ -0,0 +1,152 @@
+    parser.add_argument("--num_workers", type=int)
+    parser.add_argument("--chunk_size", type=int)
+    parser.add_argument("--scale", type=float)
+    parser.add_argument("--crop_hw", type=int, default=None)
+    args = parser.parse_args()
+
+    config = OmegaConf.load(f"{args.dir_path}/initial_config.yaml")
+
+    train_labels = sio.load_slp(config.data_config.train_labels_path)
+    max_height, max_width = get_max_height_width(train_labels)
+    val_labels = sio.load_slp(config.data_config.val_labels_path)
+    user_instances_only = False if args.user_instances_only == 0 else True
```

Ruff

Use `not ...` instead of `False if ... else True`

Replace with `not ...`

(SIM211)